packages/kotti-ui/source/index.ts

```diff
@@ -155,6 +159,7 @@ export default {
 			KtPagination,
 			KtPopover,
 			KtPopoverItem,
+			KtPopoverIconTextItem,
```

Comment:
@RueRivoli @FlorianWendelborn @carsoli  Since it is now implemented as an internal fallback, are you sure you want to expose it on the external api? Other than that code is looking good to me at this stage.

Reply:
Yes, it would make sense. But if I remove it from the export in kotti-ui/source/index.ts, the KtPopoverIconTextItem s disappear in the example of the fallback case. Just an empty popover is displayed. That's why I kept it like this.

Reply:
Yes, so import it in KtPopover directly is better